### PR TITLE
Fix stdin error on windows by using Dockerfile instead

### PR DESCRIPTION
**Pull request by julienduchesne**
JIRA CLOUDINFRA-311

### CODE REVIEW

File: docker.go [hunk 1]
```diff
@@ -125,23 +125,26 @@ func getImage() string {
 		return config.GetImageName()
 	}
 
-	args := []string{"build", "-", "--quiet", "--force-rm"}
+	var dockerFile string
+	dockerFile += fmt.Sprintln("FROM", config.GetImageName())
+	dockerFile += fmt.Sprintln(config.ImageBuild)
+
+	tempDir, _ := ioutil.TempDir(".", "tgf-docker")
```
Comment (jocgir) [inline docker.go]: Should use "" instead of "."

File: docker.go [hunk 2]
```diff
@@ -125,23 +125,26 @@ func getImage() string {
...
+	ioutil.WriteFile(fmt.Sprintf("%s/Dockerfile", tempDir), []byte(dockerFile), 0644)
```
Comment (jocgir) [inline docker.go]: No error check?

File: docker.go (hunk 1 above)
Comment (julienduchesne) [reply docker.go]: Fixed

File: docker.go (hunk 2 above)
Comment (julienduchesne) [reply docker.go]: Fixed

**Pull request merged**